aio/aio-proxy/aio_proxy/tests/test_activite_principale.py

```diff
@@ -0,0 +1,24 @@
+import parameters
+import pytest
```

Comment:
What is python best practise ? Is it to have all tests in the same folder ? Or is it to have it next to the file containing the tested code ?

Reply:
I am not aware of a specific practice. I believe it's more of a per case answer. 
In this case, we have many tests to add to the unit tests. I do think it's better to have a specific folder for them, within the project structure.